### PR TITLE
misc README updates

### PR DESCRIPTION
**Pull request by dacbd**
some quick readme updates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,18 +124,18 @@ report.
...
-| `cml send-github-check` | Return CML report as a check in GitHub                           | `<path to report> --head-sha <sha>`                         |
-| `cml pr`                | Commit the given files to a new branch and create a pull request | `<path>...`                                                 |
-| `cml tensorboard-dev`   | Return a link to a Tensorboard.dev page                          | `--logdir <path to logs> --title <experiment title> --md`   |
+| Function                  | Description                                                      | Example Inputs                                              |
+| ------------------------- | ---------------------------------------------------------------- | ----------------------------------------------------------- |
+| `cml runner launch`       | Launch a runner locally or hosted by a cloud provider            | See [Arguments](https://github.com/iterative/cml#arguments) |
+| `cml publish`             | Publicly host an image for displaying in a CML report            | `<path to image> --title <image title> --md`                |
```
Comment (0x2b3bfa0) [inline README.md:130]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -124,18 +124,18 @@ report.
...
+| `cml comment create`      | Return CML report as a comment in your GitLab/GitHub workflow    | `<path to report> --head-sha <sha>`                         |
+| `cml check create`        | Return CML report as a check in GitHub                           | `<path to report> --head-sha <sha>`                         |
+| `cml pr`                  | Commit the given files to a new branch and create a pull request | `<path>...`                                                 |
```
Comment (0x2b3bfa0) [inline README.md:133]: `cml pr create`

File: README.md (hunk 1 above)
Comment (dacbd) [reply README.md:130]: do we want to take it out completely? I know this has been pretty nicely built into `comment`. Do we want to mention the hidden alias of `asset`? It is the most used command.

Comment (tasdomas) [reply README.md:130]: We should probably not document deprecated subcommands.

**Review by tasdomas — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (0x2b3bfa0) [reply README.md:130]: I'd prefer not to mention any hidden commands

File: README.md [hunk 3]
```diff
@@ -466,72 +467,99 @@ newly-launched instance. See [Environment Variables] below for details on the
...
-                            automatically generated throwaway key will be used
-                                                                 [default: ""]
---cloud-spot                Request a spot instance                  [boolean]
---cloud-spot-price          Maximum spot instance bidding price in USD.
-                            Defaults to the current spot bidding price
-                                                               [default: "-1"]
---cloud-startup-script      Run the provided Base64-encoded Linux shell script
-                            during the instance initialization   [default: ""]
---cloud-aws-security-group  Specifies the security group in AWS  [default: ""]
+cml runner launch
```
[the remark this answers is not in the document]
Comment (dacbd) [reply README.md:488]: I see, we can strip the top part to closer match what was there previously.

File: README.md [hunk 4]
```diff
@@ -153,11 +152,12 @@ cat results.txt >> report.md
...
 if an image is an output of your ML workflow (i.e., it is produced by your
-workflow), you will need to use the `cml publish` function to include it a CML
-report. For example, if `graph.png` is output by `python train.py`, run:
+workflow), you can add `--publish` flag to include it a CML report. For example,
+if `graph.png` is output by `python train.py`, run:
 
 ```bash
-cml publish graph.png --md >> report.md
+echo "![](./graph.png)" >> report.md
+cml comment create --publish report.md
```
Comment (casperdcl) [inline README.md:160]:
```suggestion
cml comment create report.md
```

Comment (dacbd) [reply README.md:160]: This example has a local file that needs to be published?

Comment (0x2b3bfa0) [reply README.md:160]: Now `--publish=true` by default

Comment (dacbd) [reply README.md:160]: hmm, I think we should do a follow-up to either have a syntax that can prevent that on a per-link basis or only publish if the file is not tracked by git?

Comment (0x2b3bfa0) [reply README.md:160]: Why? 🤔

Comment (dacbd) [reply README.md:160]: maybe I want to control what gets uploaded and to where? why do we let you self-host your own? why can't you reference an image already in your repo without uploading it somewhere else? I think the reasons for all of these are the same.

Comment (dacbd) [reply README.md:160]: should I scrub all `--publish` references then?

Comment (0x2b3bfa0) [reply README.md:160]: > _maybe I want to control what gets uploaded and to where? why do we let you self-host your own? why can't you reference an image already in your repo without uploading it somewhere else?_

Only files liked from the specified `report.md` will be uploaded. Do you have any use case in mind where users would link to files from `report.md` and not want them to be uploaded? 😅

Comment (0x2b3bfa0) [reply README.md:160]: > _should I scrub all `--publish` references then?_

Preferably

Comment (dacbd) [reply README.md:160]: > Only files liked from the specified `report.md` will be uploaded. Do you have any use case in mind where users would link to files from `report.md` and not want them to be uploaded? 😅

linking any file that's in the repo?

Comment (0x2b3bfa0) [reply README.md:160]: E.g. the following code will only upload `/path/to/file.png`

```bash
echo '![example](/path/to/file.png)' > report.md
cml comment create report.md
```

Comment (0x2b3bfa0) [reply README.md:160]: Given that users control what they write,there should be no need for more complex solutions

Comment (dacbd) [reply README.md:160]: I would want more control than the default of uploading every file path in my markdown report, but I'm fine with filing that opinion away if no one else sees the value...

**Review by 0x2b3bfa0 — APPROVED** [review]
Looks good to me!

**Pull request merged**